src/radio.cpp

```diff
@@ -104,7 +104,13 @@ namespace bzzz
 
     bool Radio::kill()
     {
-        return channelData[RADIO_CHANNEL_SWITCH_A] <= 500;
+        return channelData[RADIO_CHANNEL_SWITCH_A] >= 1500;
+    }
+
+    float Radio::readKillSwitch()
```

Comment:
This is unnecessary. There is `radio.kill()`

Reply:
This was added to allow us to read if the kill switch has been turned off again, without it we have to reset the ESP32 every time the kill switch is triggered.